File: app/src/main/java/com/example/khughes/machewidget/CommandStatus.java
package com.example.khughes.machewidget;

import javax.annotation.Generated;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Generated("jsonschema2pojo")
public class CommandStatus {

    @SerializedName("version")
    @Expose
    private String version;
    @SerializedName("status")
    @Expose
    private Integer status;
    @SerializedName("commandId")
    @Expose
    private String commandId;

    public String getVersion() {
        return version;
    }

    public void setVersion(String version) {
        this.version = version;
    }

    public Integer getStatus() {
        return status;
    }

    public void setStatus(Integer status) {
        this.status = status;
    }

    public String getCommandId() {
        return commandId;
    }

    public void setCommandId(String id) {
        this.commandId = id;
    }

}

File: app/src/main/java/com/example/khughes/machewidget/CommandStatus.kt
package com.example.khughes.machewidget

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose
import javax.annotation.Generated

@Generated("jsonschema2pojo")
class CommandStatus {
    @SerializedName("status")
    @Expose
    var status: String? = null

    @SerializedName("commandStatus")
    @Expose
    var commandStatus: String? = null

    @SerializedName("commandId")
    @Expose
    var commandId: String? = null
}